app.py

```diff
@@ -1,34 +1,37 @@
 from time import sleep
 
-from blessed import Terminal
+from rich.layout import Layout
 from rich.live import Live
 from rich.panel import Panel
 
 from src.resources.character import Character
+from src.resources.items import AbstractDungeonItem
 from src.resources.level import Level
 
-# Used to get player input
-term = Terminal()
 
-
-def run_game(panel: Panel) -> Panel:
+def run_game(layout: Layout) -> Panel:
     """
     This function in in charge of running the game. It will call update and draw for each game object.
 
-    Layout: Layout  Holds all the rich rederables for the game. Updated with a new panel each tick.
+    Layout: Layout  Holds all the rich renderables for the game. Updated with a new panel each tick.
     """
-    player.draw(level.board)
+    dungeon_item.draw(level.board)
+    player.draw()
```

Comment:
global variables are risky business. pass them to the function as arguments.

Reply:
I believe this will be refactored later